scripts/reset-submodule

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+git submodule update --init grpc-sys/grpc
+cd grpc-sys/grpc
+git submodule update --init third_party/boringssl
```

Comment:
can we use `git submodule update --init --recursive` directly?

Reply:
Then many unused submodules get initialized, which can be slow and probably fail due to network censor.